provisioning/provisioning_info.go

```diff
@@ -16,4 +17,5 @@ type ProvisioningInfo struct {
 	Scheme        *runtime.Scheme
 	Namespace     string
 	Images        *Images
+	LabelSelector *metav1.LabelSelector
```

Comment:
We are using this selector to select Pods. So, this could be called PodLabelSelector. But, this field is in the Deployment, so this could be called the DeploymentLabelSelector. Is there a preference?

Reply:
Marking this as resolved after going with PodLabelSelector.